pulser-simulation/pulser_simulation/simconfig.py

```diff
@@ -92,6 +95,12 @@ class SimConfig:
     )
 
     def __post_init__(self) -> None:
+        if not 0.0 <= self.amp_sigma < 1.0:
+            raise ValueError(
+                "The standard deviation in amplitude (amp_sigma="
+                f"{self.amp_sigma}) must be greater than or equal"
+                " to 0. and smaller than 1."
+            )
         self._process_temperature()
```

Comment:
The lower bound should be clear, since this is a standard deviation.
I admit the upper bound is somewhat arbitrary, but since this is a normalized multiplicative factor that needs to be `<=0`, I figured that the case where that happens within 1 sigma was a reasonable upper-bound - realistically, we will never come even close to this.

Reply:
Okay, trust you on the "realistically", we could always relax this later if we need to explore uncommon regimes